source/php/Component/Icon/icon.blade.php

```diff
@@ -1,8 +1,11 @@
 <!-- icon.blade.php -->
 @if ($icon)
     <{{ $componentElement }} class="{{ $class }}" {!! $attribute !!}>
-        @if ($isSvg)
+        @if ($isSvgLink)
```

Comment:
May we split each case in it's own named views? They are small, but this might give us some enhanced readability.